src/backend/egl/context.rs

```diff
@@ -16,6 +16,8 @@ pub struct EGLContext {
     config_id: ffi::egl::types::EGLConfig,
     pixel_format: PixelFormat,
 }
+unsafe impl Send for EGLContext {}
+unsafe impl Sync for EGLContext {}
```

Comment:
We need this for `EGLSurface` and the egl api does support it (it is internally locked).